src/lib/convenience/AppendContextElementResponse.cpp

```diff
@@ -120,6 +122,71 @@ void AppendContextElementResponse::release(void)
 {
   LM_T(LmtRelease, ("Releasing AppendContextElementResponse"));
 
-  contextResponseVector.release();
+  contextAttributeResponseVector.release();
   errorCode.release();
 }
+
+
+
+/* ****************************************************************************
+*
+* AppendContextElementResponse::fill - 
+*
+* NOTE
+* This method is used in the service routine of 'POST /v1/contextEntities/{entityId::id} et al.
+* Only ONE response in the vector contextElementResponseVector of UpdateContextResponse is possible.
+* FIXME P7: Is this true? Only ONE response?
```

Comment:
Yeah, there is no responses with more than one in the vector in the wiki and I also cannot find any circumstance with a response of more than one.